official/projects/cots_detector/crown_of_thorns_starfish_detection_pipeline.ipynb

```diff
@@ -64,7 +64,7 @@
     "id": "jDiIX2xawkJw"
```

Comment:
<p>This needs to be updated to match the new code - possibly removed or reduced a bit?</p><p> </p><p>At the very least the line numbers don't match (and aren't enabled by default in colab either), and it looks like it applies to the code a couple of cells down.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/models/pull/10640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
<p>Done.</p>